hypertools/tools/cluster.py

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env python
 
 from sklearn.cluster import KMeans
-from sklearn.decomposition import PCA
+import warnings
 import numpy as np
 from .._shared.helpers import *
+from .reduce import reduce as reducer
+from .align import align as aligner
+from .normalize import normalize as normalizer
 
-def cluster(x, n_clusters=8, ndims=None):
+def cluster(x, n_clusters=8, ndims=None, model='IncrementalPCA',
```

Comment:
I decided not to include kwargs for `align_params`, `cluster`, and `cluster_params` since there is currently nothing implemented to handle these arguments. I changed `model` and `model_params` to `reducer` and `reducer_params`, respectively; I avoided using `reduce` since it is a Python keyword.